### PR TITLE
Added Transfer money implementation using TDD and Pair Programming

### PR DESCRIPTION
**Pull request by biniama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,5 @@ Expected Response: GRANT
 Grant permissions to the user to be able to login and create tables  
 `ALTER USER moneytransfer WITH LOGIN;`   
 Expected Response: ALTER  
+
+https://code-with-me.jetbrains.com/6vUuhZCo0iVqPLtTuf_uFw#p=IU&fp=629F8212CD968BEA6C4A8B525CB0059699B4152533987920149CDB5E0208654D
```
Comment (biniama) [inline README.md:43]: Why did you add this?

File: src/main/java/edu/act/moneytransfer/controllers/TransferController.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import edu.act.moneytransfer.repositories.TransferRepository;
+import edu.act.moneytransfer.services.TransferService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TransferController {
```
Comment (biniama) [inline src/main/java/edu/act/moneytransfer/controllers/TransferController.java:14]: Nice work!

**Review by biniamkefale — APPROVED** [review]
TDD and Pair Programming. Nice work!
